src/components/TopNav.astro

```diff
@@ -0,0 +1,9 @@
+---
+const { parent_nav_path, parent_nav_text} = Astro.props;
+import previousNavIcon from '../assets/images/previous-nav-icon.svg'
+const url = `${import.meta.env.BASE_URL}${parent_nav_path}`
+
+---
+<div class="display-flex margin-bottom-2" style="width:100%">
+    <img src={previousNavIcon} alt="left Arrow icon for navigation back to Policies and Audits"><a class="usa-link" href={`${import.meta.env.BASE_URL}${parent_nav_path}`}>Return to the {parent_nav_text}</a>
```

Comment:
We can talk about how these links will work when Kristen is back, but in general we don't want screen readers to see the icons. The code in https://designsystem.digital.gov/components/pagination/ is a good example — they implement the arrow icon with `aria-hidden="true" role="img"` to prevent screen readers from see the icon.

Reply:
sure, per our discussion, I keep current top nav using link (matching mockup) for now, once we talked to Kirsten, we can update the TopNav based on final decision, thanks!